modules/apps/foundation/portal-security/portal-security-antisamy/src/main/resources/META-INF/resources/sanitizer-configuration.xml

```diff
@@ -490,6 +490,7 @@ http://www.w3.org/TR/html401/struct/global.html
 
 	<allowed-empty-tags>
 		<literal-list>
+			<literal value="img" />
```

Comment:
this is very unfortunate. 
It looks like this is not supported and we will need to whitelist the tags. 
I will send a new pull with what seem to be the default whitelisted tags.